src/test/resources/features/Meeds/MeedsUsr_Mng.feature

```diff
@@ -1,17 +1,14 @@
-# language: en
+@users
 Feature: Users Management
   As an administrator
   I want to check that i can manage the users
 
-  @ignored
   Scenario: [USR_MNG-1.1] Users Grid
     Given I am authenticated as admin
 
-    When I create the first random user if not existing
-    And I refresh the page
-    And I create the second random user if not existing
-    And I refresh the page
-    And I search for second created user
+    When I create the first random user if not existing, no wait
+    And I create the second random user if not existing, no wait
```

Comment:
The user is used in search just after, thus we need to wait a little time until it's indexed
```suggestion
    And I create the second random user if not existing
```

Reply:
done